src/tasks.md

```diff
@@ -57,30 +57,31 @@ You can list all the `kwctl` options and subcommands by running the following co
 
 ```shell
 $ kwctl --help
-kwctl 0.2.4
-Flavio Castelli <fcastelli@suse.com>:Rafael Fernández López <rfernandezlopez@suse.com>
+kwctl 0.2.5
+Kubewarden Developers <kubewarden@suse.de>
 Tool to manage Kubewarden policies
 
 USAGE:
-    kwctl [FLAGS] <SUBCOMMAND>
+    kwctl [OPTIONS] <SUBCOMMAND>
 
-FLAGS:
-    -h, --help       Prints help information
-    -V, --version    Prints version information
+OPTIONS:
+    -h, --help       Print help information
     -v               Increase verbosity
+    -V, --version    Print version information
 
 SUBCOMMANDS:
-    annotate       Adds Kubewarden metadata to a WebAssembly module
-    completions    Generates shell completions
-    help           Prints this message or the help of the given subcommand(s)
-    inspect        Inspects Kubewarden policy
-    manifest       Scaffolds a Kubernetes resource
+    annotate       Add Kubewarden metadata to a WebAssembly module
+    completions    Generate shell completions
+    digest         Fetch the digest of its OCI manifest
+    help           Print this message or the help of the given subcommand(s)
+    inspect        Inspect Kubewarden policy
     policies       Lists all downloaded policies
     pull           Pulls a Kubewarden policy from a given URI
     push           Pushes a Kubewarden policy to an OCI registry
     rm             Removes a Kubewarden policy from the store
     run            Runs a Kubewarden policy from a given URI
-    verify         Verifies a Kubewarden policy from a given URI using Sigstore
+    scaffold       Scaffold a Kubernetes resource or configuration file
```

Comment:
It’d be better to clarify which particular configuration file this scaffolds. 

- Is it ONLY a Kubernetes configuration file? 
- Is it any configuration file that can be scaffolded but passed as a separate parameter? 

```suggestion
    scaffold       Scaffolds a Kubernetes resource or configuration file
```

Reply:
Agree that it could be more verbose, but once one runs the command, it will show the subcommands:
```console
$ kwctl scaffold
error: 'kwctl scaffold' requires a subcommand but one was not provided

USAGE:
    kwctl scaffold <SUBCOMMAND>

For more information try --help
vic@viccuad2 2 ~$ kwctl scaffold --help
kwctl-scaffold
Scaffold a Kubernetes resource or configuration file

USAGE:
    kwctl scaffold <SUBCOMMAND>

OPTIONS:
    -h, --help    Print help information

SUBCOMMANDS:
    help                   Print this message or the help of the given subcommand(s)
    manifest               Output a Kubernetes resource manifest
    verification-config    Output a default Sigstore verification configuration file

```

Reply:
Okay, then contextually it makes sense.